src/frontend/Komponenter/Behandling/VedtakOgBeregning/Barnetilsyn/AktivitetSelect.tsx

```diff
@@ -28,23 +40,28 @@ const valgbareAktivitetstyper = [
 ];
 
 const AktivitetSelect: React.FC<Props> = ({
+    periodetype,
     aktivitet,
     oppdaterUtgiftsperiodeElement,
     erLesevisning,
-    erMidlertidigOpphør,
     feil,
 }) => {
-    const utledLesevisningVerdi = () => {
-        if (aktivitet) return utgiftsperiodeAktivitetTilTekst[aktivitet];
-        if (erLesevisning && erMidlertidigOpphør) return '';
-        return 'Ukjent';
-    };
+    const skalIkkeVelgeAktivitet: boolean = !periodetype || erOpphørEllerSanksjon(periodetype);
+    if (skalIkkeVelgeAktivitet) {
+        return (
+            <AktivitetKolonne
+                medPadding={!erLesevisning && periodetype !== EUtgiftsperiodetype.SANKSJON_1_MND}
+            >
+                <BodyShort size={erLesevisning ? 'small' : 'medium'}>-</BodyShort>
```

Comment:
Sikker på at vi vil ha med denne? Personlig foretrekker jeg det helt clean

Reply:
Såg at det ikke var noe `-` nå i skissene og Lars ønsket ju å fjerne valget av barn for opphør og sanksjon så tar tilbake dette :)